packages/block-library/src/query-loop/index.php

```diff
@@ -44,25 +54,25 @@ function render_block_core_query_loop( $attributes, $content, $block ) {
 		if ( isset( $block->context['query']['perPage'] ) ) {
```

Comment:
This is not from this PR but I just noticed that. We check twice for  `$block->context['query']['perPage']`. We can combine them.

Reply:
Fixed